connectors/sources/confluence.py

```diff
@@ -37,12 +37,12 @@
 SPACE_QUERY = "limit=100"
 ATTACHMENT_QUERY = "limit=100&expand=version"
 CONTENT_QUERY = (
-    "limit=100&expand=children.attachment,history.lastUpdated,body.storage,space"
+    "limit=50&expand=children.attachment,history.lastUpdated,body.storage,space"
```

Comment:
Why change the limit to 50?

Reply:
We are now using search endpoint and the max allowed results for  search endpoint is 50.